examples/chip-tool/args.gni

```diff
@@ -16,6 +16,8 @@ import("//build_overrides/chip.gni")
 
 import("${chip_root}/config/standalone/args.gni")
 
+chip_build_example_creds = true
```

Comment:
This flag is true by default, what does this do?

Reply:
Was getting build errors with the chip-tool since it was using it.